lib/routes/gcores/category.ts

```diff
@@ -126,6 +128,7 @@ async function handler(ctx) {
                     description: cover + content,
                     link: articleUrl,
                     guid: articleUrl,
+                    author: item.author,
```

Comment:
Since the API response https://github.com/DIYgod/RSSHub/blob/21f994f7c182378db544c443cd709cfc5477e8b9/lib/routes/gcores/category.ts#L81
returns the author id.
If I were you, I would rather make another cached API request for `nickname`.

Reply:
Ah, I see. This is what I looked for! And I didn't notice this because now it requires a long param to include all those metadata. A browser GET request header is something like
```
https://www.gcores.com/gapi/v1/articles/180031?include=category,user,user.role,tags,entities,entries,similarities.user,similarities.djs,similarities.category,collections,operational-events.giveaways.winners,operational-events.public-candidates,portfolios&preview=1
```

I'll try to use this instead.

Reply:
This does work better and no need to use css selector. I'll clean up codes and push again later.